book/website/collaboration/new-community/new-recruitment.md

```diff
@@ -0,0 +1,555 @@
+Having a specific timeline outlined and communicated from the start of the recruitment process is important not only for hiring teams but also for applicants. 
+
+
+### Before advertising 
+
+
+
+- [ ] Ensure effective communication  between hiring manager and recruitment team - know what your recruitment team is already doing (for example  where roles will be advertised) to avoid working in silos
+- [ ] Writing the job description (JD)
+    * Detail what the person will be doing
+         * Recommend 10-12 bullet points
+         * Focus on the tasks to be completed rather than _how_ the task is completed
+    * Write in plain English, removing any jargon and abbreviations
+    * Consider unconscious bias against all characteristics - often unconscious bias moves beyond the nine protected characteristics and into dimensions such as education level, socioeconomic status, geography, and hobbies.
+        * Avoid superlatives
+        * Check for accessibility of advert using a screen reader
+        * Keep mandatory skills to a minimum and thoroughly consider whether all criteria are essential
+        * Avoid asking for mandatory travel, weekend work and overtime, if possible
+    * Remove all biased language in the job advert
+        * Consider using a gender decoder to identify words/phrases that may imply a specific gender and re-phrase if needed
+    * Provide clear guidelines & deadlines plus instructions & contacts for people requiring reasonable adjustments 
+        * Provide transparency over interview structure and application assessment criteria where possible 
+    * Include relevant policies clearly in the job advert 
+    * Be transparent about the compensation package
+        * Provide a starting salary or salary range 
+        * Include any statutory or enhanced benefits your organisation may offer such as paternity, maternity and parental leave and/or benefits such as time off for dependents, and annual leave entitlement. 
+    * Directly reference your organisational EDI commitment
+        * Include your organisational Diversity Statement where available.
+        * Include your organisational Values (if applicable).
+        * Include you team's personal commitment to Equity, Diversity and Inclusive (if applicable). 
+- [ ] Post job advert 
+    * Consider the best places to advertise to reach a diverse pool of candidates
+        * See “ Resources” section for suggestions of advertising venues
+        * If certain demographics are often missing from your pool, utilise spaces that target those groups
+        * Consider reviewing the diversity of the pool attained from previous, similar advertising and seek specific advertising to reach other groups that were underrepresented 
+    * Use a gender decoder to review the advert after you have created it
+    * Consider digital accessibility on social media and in job adverts
```

Comment:
You could link to some of the accessibility work the other guides have in them, such as the alt-text guide, here